core/settings.py

```diff
@@ -168,27 +167,26 @@
 CELERY_BROKER_URL = os.getenv("CELERY_BROKER_URL")
 CELERY_RESULT_BACKEND = "django-db"
 CELERY_CACHE_BACKEND = "django-cache"
-CELERY_TIMEZONE = "Europe/Berlin"
+CELERY_TIMEZONE = "Asia/Kathmandu"
```

Comment:
Change the database timezone too if you are changing the celery beat timezone to maintain consistency